src/Components/IndexApp/IndexApp.js

```diff
@@ -0,0 +1,38 @@
+import React,{Fragment} from 'react'
+import '../IndexApp/IndexApp.scss'; 
+import Logo from '../Logo/Logo';
+import Button from '../Button/Button';
+import { useHistory } from "react-router-dom";
+
+
+
+function IndexApp () {
+    let history = useHistory()
+
+    function handleClickOrden() {
+
+        history.push('/orden')
+    }
+    function handleClickCocina() {
+
+        history.push('/cocina')
+    }
```

Comment:
consistencia si vaz a usa function ó arrow function
```suggestion
    function handleClickOrden() {

        history.push('/orden')
    }
    function handleClickCocina() {

        history.push('/cocina')
    }
```